Arrangement_on_surface_2/include/CGAL/Arr_dcel_base.h

```diff
@@ -1079,8 +1089,11 @@ class Arr_dcel_base {
   Vertex* new_vertex()
   {
     Vertex* v = vertex_alloc.allocate(1);
-
+#ifdef CGAL_CXX11
+    std::allocator_traits<Vertex_allocator>::construct(vertex_alloc,v, Vertex());
```

Comment:
This invokes the default constructor and create `Vertex()`, and then it invokes the copy- or move- constructor, and with `Vertex()` as argument.

The following is better, calling the constructor once, with zero-argument.
```C++
std::allocator_traits<Vertex_allocator>::construct(vertex_alloc,v);
```